### PR TITLE
Update jquery.socialfeed.js

### PR DESCRIPTION
**Pull request by HartLarsson**
I've added for "RSS feed type" the first image search instead of no image in case of mediaGroups empty (some blogs not provide that parameter in the feed).
I've cheked also that in wordpress the element.mediaGroups[0].contents[0].url provide the Gravatar logo and not the first media image. so i switched to element.mediaGroups[0].contents[1].url
Probably i will release a better solution for mediaGroups in few days

### CODE REVIEW

**Comment by pavelk2** [comment]
@HartLarsson 
please do not leave console.log() in code so the console stays clean.

**Comment by HartLarsson** [comment]
yes sorry i cleaned the code :)
there is another problem now Pavel, seems Google Feed API was def. removed and deprecated.

File: js/jquery.socialfeed.js [hunk 1]
```diff
@@ -737,7 +736,7 @@ if (typeof Object.create !== 'function') {
                         var post = {};
 
                         post.id = index;
-                        post.dt_create= moment(element.publishedDate, 'ddd, DD MMM YYYY HH:mm:ss ZZ', 'en');
+						post.dt_create= moment(element.publishedDate);
```
Comment (yp28) [inline js/jquery.socialfeed.js:739]: @HartLarsson, got some weird indents in here as well.

Comment (HartLarsson) [inline js/jquery.socialfeed.js:739]: very strange i use notepad++

File: js/jquery.socialfeed.js [hunk 2]
```diff
@@ -720,8 +720,7 @@ if (typeof Object.create !== 'function') {
 
                 getData: function(url) {
                     var limit = '&num='+ options.rss.limit,
-                      request_url = Feed.rss.api + limit + '&q=' + encodeURIComponent(url);
-
+                    request_url = Feed.rss.api + limit + '&q=' + encodeURIComponent(url);
```
Comment (yp28) [inline js/jquery.socialfeed.js:723]: And here should be an indent. It should align with limit since it's in the same var declaration.

**Comment by Webarkitekt** [comment]
this is linked to issue #147

**Comment by pavelk2** [comment]
@Webarkitekt 
As I understood we have @HartLarsson creating a pull request to fix some bugs. He is improving his pull request here. Why do we close this issue upon the pull request? Will we be able to still merge the pull request?

**Comment by Webarkitekt** [comment]
Because all the rss part has been rewrited here https://github.com/pavelk2/social-feed/commit/54a9d4583ebec055138d39b25b325a3726bf0302

**Pull request closed without merging**